specs/components/form/base.specs.js

```diff
@@ -204,9 +204,9 @@ describe('Common Form features', function() {
             ];
 
             expect(this.vm.serialize()).to.eql({
-                input: '',
+                input: undefined,
```

Comment:
Same as above, simply a backport.

Actually, we are testing the `undefined` value explicitely here because it matters. `null` can't and shouldn't be used because of this:
```javascript
JSON.stringify({key: null})
"{"key":null}"
JSON.stringify({key: undefined})
"{}"
```
We don't want `undefined` value to be serialized.